src/modules/upcoming-events.module/fields.json

```diff
@@ -1 +1,12 @@
-[]
+[
+  {
+    "label": "Page title",
+    "name": "page_title",
+    "type": "text",
+    "required": false,
+    "locked": false,
+    "allow_new_line": false,
+    "show_emoji_picker": false,
```

Comment:
I'd set this to `true` in case someone wants to use an emoji as an icon or something.